reconcile/queries.py

```diff
@@ -704,10 +707,26 @@ def get_app_interface_sql_queries():
 """
 
 
-def get_saas_files():
+def get_saas_files(saas_file_name='', env_name=''):
```

Comment:
So, when the cli runs:
```
qontract-reconcile --config config.toml --dry-run openshift-saas-deploy --saas-file-name ""
```
Should we find no saas files or all saas files? As of now, we find all saas files.
Not a big deal, since we have the cli under our control... just setting the expectations here.
If we want "no saas files", the default here (and everywhere else) should be `None` and the condition `if saas_file_name is not None`.

Reply:
the default should be all saas files, and that is indicated by not passing `--saas-file-name`.

similar to `kubectl get pods` vs `kubectl get pod <name>`.

Reply:
i'm going to merge this and we can discuss further. we will have many more iterations :)